package.json

```diff
@@ -127,7 +127,7 @@
     }
   },
   "engines": {
-    "node": "13.x",
-    "npm": ">= 6.0.0"
+    "node": ">=14.15.0",
```

Comment:
@Zeko369  were you suggesting @megajon and @tomnoland remove this "engines" section entirely?

They both suggested not supporting Node 13, so would it be best to fix these to a Node 14 compatible version, or remove the section, [like you suggested on Discord](https://discord.com/channels/633401573002969128/633826607617933353/817832134387040318)?

Reply:
I noticed that the master branch had the engine set to node 14 so I copied those settings to my sendEmail branch and I no longer get the error.

Reply:
Another problem I'm having is that when I run the client side server from my sendEmail branch I get an error stating that a certain module can't be resolved. I noticed that my sendEmail branch is missing a few directories under client/src/ that are present in the master branch, which seems to work fine when I run both the client and backend servers. This will be my first time merging code into an open source project so I'm guessing the directories are missing from my sendEmail branch because they were created after I created the branch. I'm assuming everything will work properly once the code is merged, but I just wanted to check to make sure I'm not missing something.